sys/sys.txt

```diff
@@ -937,6 +968,8 @@ renameat2_flags = RENAME_EXCHANGE, RENAME_NOREPLACE, RENAME_WHITEOUT
 flock_op = LOCK_SH, LOCK_EX, LOCK_UN, LOCK_NB
 seccomp_op = SECCOMP_SET_MODE_STRICT, SECCOMP_SET_MODE_FILTER
 seccomp_flags = 0, SECCOMP_FILTER_FLAG_TSYNC
+statx_flags = STATX_ATTR_COMPRESSED, STATX_ATTR_IMMUTABLE, STATX_ATTR_APPEND, STATX_ATTR_NODUMP, STATX_ATTR_ENCRYPTED, KSTAT_ATTR_FS_IOC_FLAGS, STATX_ATTR_AUTOMOUNT
```

Comment:
I have some problems about this line.
As the [Manual](http://man7.org/linux/man-pages/man2/statx.2.html) said, statx_flag used in syscall is constructed by `AT_EMPTY_PATH`, `AT_NO_AUTOMOUNT` , `AT_SYMLINK_NOFOLLOW` etc. , not the values in this line.
`STATX_ATTR_*` is used in the returned information `stx_attributes`.